packages/comet-uswds/src/components/text-area/text-area.tsx

```diff
@@ -1,6 +1,6 @@
 import classnames from 'classnames';
 import React, { ChangeEventHandler } from 'react';
-import { ValidationStatus } from 'src/utils/types';
+import { ValidationStatus } from '../utils/types';
 import FormGroup from '../form-group';
 
 export interface TextAreaProps {
```

Comment:
Maybe add an optional character counter and an optional max character prop?

Reply:
This component includes all props supported by an HTML input, based on this line of code: `TextAreaProps & JSX.IntrinsicElements['textarea']`, so the maxLength prop could be added as needed.

And I think adding Character Count as a separate component as needed would be best.